docs/av-fetch.1.md

```diff
@@ -0,0 +1,13 @@
+# av-pr-fetch 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-fetch - Fetch latest repository state from github.
```

Comment:
```suggestion
av-pr-fetch - Fetch latest pull request state from GitHub
```

Reply:
GitHub is generally stylized as `GitHub` (only `github` in code if the first letter must be lowercase).